### PR TITLE
log the request parameters of GET requests

### PR DESCRIPTION
**Pull request by danielyhuang**
**Affects:** \<api-audit-version-number>.

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java [hunk 1]
```diff
@@ -114,12 +117,17 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         } catch (MimeTypeParseException e) {
             LOGGER.error("Invalid MIME Type detected. Request MIME type=" + httpServletRequest.getContentType() + ". Response MIME Type=" + bufferedResponse.getContentType());
         } finally {
+            String parameters = Collections.list(request.getParameterNames()).stream()
```
Comment (nireeshT) [inline src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java:120]: request.getParameterNames() = null, cause NPE

File: src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java [hunk 2]
```diff
@@ -114,12 +117,17 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
...
+                            .map(p -> p + ":" + Arrays.asList( request.getParameterValues(p)) )
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java:121]: please put a null check for request.getParameters MapUtils.isNotEmpty...

**Review by nireeshT — CHANGES_REQUESTED** [review]
Review comments

**Review by rvema — CHANGES_REQUESTED** [review]
please address the minor review comment

File: src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java (hunk 1 above)
Comment (danielyhuang) [reply src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java]: add a null check for request.getParameters

File: src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java (hunk 2 above)
Comment (danielyhuang) [reply src/main/java/com/capitalone/dashboard/logging/LoggingFilter.java]: added

**Review by nireeshT — APPROVED** [review]
LGTM

**Review by rvema — APPROVED** [review]
lgtm

**Pull request merged**